src/bin/inx-chronicle/migrations/migrate_20230202.rs

```diff
@@ -0,0 +1,122 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use async_trait::async_trait;
+use chronicle::{
+    db::{collections::OutputCollection, MongoDb, MongoDbCollectionExt},
+    types::stardust::block::output::{AliasId, NftId, OutputId},
+};
+use futures::TryStreamExt;
+use mongodb::{bson::doc, options::IndexOptions, IndexModel};
+use serde::Deserialize;
+
+use super::Migration;
+
+pub struct Migrate;
+
+#[async_trait]
+impl Migration for Migrate {
+    const VERSION: &'static str = "20230202";
```

Comment:
How can a date be a version? Why can't we just use a simple version number that gets incremented with each migration?

Reply:
Technically it's not a date, it's a string. Serious answer: I wanted something that could could be easily understood and would tell you when it happened. As Jochen pointed out, we could still do two (or more) migrations in one day, but let's be honest, that will never happen.